tests/wpt/metadata/html/dom/interfaces.html.ini

```diff
@@ -8658,6 +8658,12 @@
   [Location interface: window.location must inherit property "reload" with the proper type (2)]
     expected: FAIL
 
+  [Location interface: existence and properties of interface prototype object]
+    expected: FAIL
+
+  [Stringification of window.location]
+    expected: FAIL
```

Comment:
Right, both of these tests are expecting an "[object ..." style stringification that's been invalidated.